Ratings_MVP/Ratings_MVP/SearachFilmScreen/Cells/MovieCell.swift

```diff
@@ -11,6 +11,18 @@ final class MovieCell: UITableViewCell {
         static let backgoundColorName = "backgroundColor"
         static let textColorName = "textColor"
         static let startUrlText = "https://image.tmdb.org/t/p/w300"
```

Comment:
```suggestion
        static let startUrlText = "https://image.tmdb.org/t/p/w300"
```
все что с запросами в апи сервис перенеси